File: src/scanners/RetroarchFlatpakSource.js
const RetroarchSource = require("./RetroarchSource.js");
const RetroarchFlatpakGame = require("../games/RetroarchFlatpakGame.js");

const USER_DIR = process.env["HOME"];

class RetroarchFlatpakSource extends RetroarchSource{

	static name = "Retroarch (Flatpak)";
	static gameClass = RetroarchFlatpakGame;

	configPath = `${USER_DIR}/.var/app/org.libretro.RetroArch/config/retroarch/playlists`;

}

module.exports = RetroarchFlatpakSource;

File: gali/src/sources/retroarch/flatpak/retroarch_flatpak_source.py
from gali.sources.retroarch.flatpak.retroarch_flatpak_game import RetroarchFlatpakGame
from gali.sources.retroarch.retroarch_source import RetroarchSource
from gali.utils.locations import HOME


class RetroarchFlatpakSource(RetroarchSource):

    name: str = "Retroarch (Flatpak)"
    game_class: type[RetroarchFlatpakGame] = RetroarchFlatpakGame
    config_path: str = f"{HOME}/.var/app/org.libretro.RetroArch/config/retroarch/retroarch.cfg"